doit.go

```diff
@@ -40,26 +44,115 @@ func (c *LiveConfig) GetGodoClient() *godo.Client {
 	return godo.NewClient(oauthClient)
 }
 
+func sshConnect(user string, host string, method ssh.AuthMethod) (err error) {
+	sshc := &ssh.ClientConfig{
+		User: user,
+		Auth: []ssh.AuthMethod{method},
+	}
+	conn, err := ssh.Dial("tcp", host, sshc)
+	if err != nil {
+		return err
+	}
+
+	session, err := conn.NewSession()
+	if err != nil {
+		return err
+	}
+
```

Comment:
nitpick: always keep the `defer close` block just next to where the resource is acquired:

```go
session, err := conn.NewSession()
if err != nil {
    return err
}
defer session.Close() // right next to the if-err block
```